### PR TITLE
Adding support for OSX

### PR DESCRIPTION
**Pull request by plivesey**

### CODE REVIEW

**Comment by plivesey** [comment]
@kevinvitale @nicksnyder

File: build.sh [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
 time xcodebuild clean test \
     -project ConsistencyManager.xcodeproj \
     -scheme ConsistencyManager \
+    -sdk macosx10.11 \
+    -derivedDataPath $DERIVED_DATA \
+    | tee build.log \
+    | xcpretty &&
+rm -rf $DERIVED_DATA &&
+time xcodebuild clean build \
+    -project LayoutKit.xcodeproj \
```
Comment (nicksnyder) [inline build.sh]: This isn't LayoutKit :)

Comment (plivesey) [reply build.sh]: You caught me... :(

File: build.sh [hunk 2]
```diff
@@ -1,11 +1,27 @@
 #!/bin/sh
+# Builds all targets and runs tests.
+
+DERIVED_DATA=${1:-/tmp/LayoutKit}
```
Comment (nicksnyder) [inline build.sh]: might want to change this default here too :)

**Comment by nicksnyder** [comment]
lgtm

**Pull request merged**